examples/stackdriver_agent_roles/CLOUDSHELL_INSTRUCTIONS.txt

```diff
@@ -0,0 +1 @@
+To set up a service account to write to Stackdriver, follow the above instructions to set your project ID, run `terraform init && terraform apply`, and enter the email address of the service account.
```

Comment:
Instead of referencing the "above instructions" I think it'd be better to actually include the instructions in the tutorial.